app/helpers/application_helper.rb

```diff
@@ -1,2 +1,16 @@
 module ApplicationHelper
+
+  def sidebar_active_link(active_link)
+    css_class = ''
```

Comment:
Not elegant way to handle selected links

Refere following linsk

https://github.com/vigetlabs/nav_lynx
http://stackoverflow.com/a/3708730/517483

You may need to modify routes to properly use helpers suggested in above links.